msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -11,10 +11,15 @@ ms.topic: overview
 You can allow orgs (specifically Teams admins) to customize some aspects of your Microsoft Teams app. Some possible examples include:
 
 * Changing the app's accent color to match the org's brand.
-* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. (Note: Users adding a connector to a chat will still see the original app name, *Contoso*).
+* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see. 
+   > [!NOTE] 
+   > Users adding a connector to a chat will still see the original app name, *Contoso*.
```

Comment:
```suggestion
* Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see.  (Note: Users adding a connector to a chat will still see the original app name, *Contoso*.)
```
Suggest reverting this to an inline note to reduce visual noise. In general, note stylings should be reserved for things that affect the entire feature. I am not sure this does. Also, does this apply to adding a connector to a channel?